modules/core/include/opencv2/core/persistence.hpp

```diff
@@ -444,7 +444,9 @@ class CV_EXPORTS_W FileStorage
     /// @overload
     CV_WRAP void write(const String& name, const String& val);
     /// @overload
-    CV_WRAP void write(const String& name, InputArray val);
+    CV_WRAP void write(const String& name, const Mat& val);
```

Comment:
does it mean that we loose ability to write `UMat` or `std::vector<float>` into the FileStorage?
Can we just add the second signature `CV_WRAP void write(const String& name, const std::vector<String>& val);` without changing the first one?

Reply:
for `UMat`, yes. Python tuples are converted into a Mat, so a `std::vector<float>` does not exists in the context of wrapping.

just adding the `vector<String>` version does not work because the implicit Mat to std::vector conversion:
```
pyopencv_generated_types.h:10611:41: error: call of overloaded ‘write(cv::String&, cv::Mat&)’ is ambiguous
```